cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/compute_fleet_status/update_compute_fleet_status.erb

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+cat << EOF > update-compute-fleet-status.py
```

Comment:
same consideration done for the other script

Reply:
Changed